src/ebay-menu/menu.tsx

```diff
@@ -87,7 +86,7 @@ const EbayMenu: FC<Props> = ({
     }, [checkedIndexes])
 
     return (
-        <span {...rest} className={classNames(className, 'menu')}>
+        <span {...rest} className={className}>
```

Comment:
The logic for className changed completely here. Not sure if it's intended but I think there might be a mistake to have the `menu` as default classname (see line 28)

If someone uses `<EbayMenu>` without classname, then the output will generate `<span class="menu">`, everything's fine.

However, once the user add custom className like `<EbayMenu className="menu-cancellation">`, the output will lose its `menu` class name (output: `<span class="menu-cancellation">`). 

Not sure if that's what we want since class `menu` is still required for Skin menu.